js/modules/loki_rpc.js

```diff
@@ -113,6 +113,13 @@ const rpc = (
   const portString = port ? `:${port}` : '';
   const url = `${address}${portString}${endpoint}`;
   // TODO: The jsonrpc and body field will be ignored on storage server
+  if (params.pubKey) {
+    // eslint-disable-next-line no-param-reassign
+    params = {
+      ...params,
+      pubKey: getStoragePubKey(params.pubKey),
+    };
+  }
```

Comment:
It might be less efficient, but how about simply `params.pubKey = getStoragePubKey(params.pubKey)`?

Reply:
Have to keep the if check to avoid undefined error but not sure why I was doing the `...` thing

Reply:
Actually, reverting this because we need to make sure we aren't modifying the params outside the function call